conjure-java-jaxrs-client/src/test/java/com/palantir/conjure/java/client/jaxrs/TracerTest.java

```diff
@@ -54,18 +54,25 @@ public void before() {
     }
 
     @Test
-    public void testClientIsInstrumentedWithTracer() throws InterruptedException, IOException {
+    public void testClientIsInstrumentedWithTracer() throws InterruptedException {
         server.enqueue(new MockResponse().setBody("\"server\""));
         OpenSpan parentTrace = Tracer.startSpan("");
 
+        AtomicInteger called = new AtomicInteger();
         Tracer.subscribe(TracerTest.class.getName(), span -> {
-            assertThat(span.getOperation(), either(is("acquireLimiter")).or(is("GET /{param}")));
+            if (span.type().equals(SpanType.CLIENT_OUTGOING)) {
+                assertThat(span.getOperation(), is("GET /{param}"));
+            } else {
+                assertThat(span.getOperation(), is("acquireLimiter"));
+            }
+            called.getAndIncrement();
```

Comment:
nit: I'd prefer to push the spans into a collection and verify after. As written this will pass if the `acquireLimiter` span is created twice (also true of the previous implementation).

Reply:
Ok